attributes/default.rb

```diff
@@ -29,6 +29,7 @@
 default['orchestrator']['config']['SlaveStartPostWaitMilliseconds'] = 1000
 default['orchestrator']['config']['UnseenInstanceForgetHours'] = 240
 default['orchestrator']['root_db_pass'] = 'snakeoil'
```

Comment:
I'd add something in the README about this new attribute as well and explain that 'if you set this to false, you better be setting the `MySQLOrchestratorHost` config to something. Or else that defaults to `localhost` and you now have a broken orchestrator